### ISSUE TITLE
Disco: Styles: Variations

### ISSUE THREAD

**Issue by matiasbenedetto** [Automattic/themes#6220]

**Comment by beafialho** [comment]
We still don't have any style variations for Club. Maybe we can add them after the theme has launched?

**Comment by jffng** [comment]
@jeffikus should we move this out of the to-do column, as there are no variations to create for now?

**Comment by jeffikus** [comment]
Yes, this is the ideal thing for the post launch tasks! Thanks for flagging this @jffng !
